### DISCUSSION TITLE
Is possible to enable the colors of the methods on the side panel?

### DISCUSSION THREAD

**Discussion by carmolim** [Kong/insomnia#6371] [Help]
With the last update the list of requests changed the method appearance, can the colors be put back? It isn't the cleanest look but the colors helped a lot to find the right request.

<img width="244" alt="image" src="https://github.com/Kong/insomnia/assets/2863161/806c24c8-8283-40c0-9c9c-0100e1a8e9a7">

**Comment by metal-gabe** [comment]
Agreed, the visual colors and differences were a good way to pick out which request I needed to look at/find. Especially when differentiating REST `POST` requests from those that are GQL `POST` requests.

**Comment by carmolim** [comment]
There's a PR opened to fix it https://github.com/Kong/insomnia/pull/6376
